app/controllers/arclight/repositories_controller.rb

```diff
@@ -6,5 +6,12 @@ class RepositoriesController < ApplicationController
     def index
       @repositories = Arclight::Repository.all
     end
+
+    def show
+      repository = Arclight::Repository.find_by!(slug: params[:id])
+      redirect_to search_action_path(
```

Comment:
I think that's how I wrote it out; I guess I'm not super opinionated about it, but the intent is in a future work cycle we may want to add a new set of behaviors for the repository show page (e.g., showing a list of "highlighted" collections as shown in in the mockups, etc.)